### ISSUE TITLE
Add dotnet 2.2 runtime support to default deployments

### ISSUE THREAD

**Issue by csantanapr** [apache/incubator-openwhisk#4177]
Title: Add `dotnet:2.2` runtime support to default deployments

### 1. Add `dotnet:2.2` to core ansible/vagrant deploy
- Add `dotnet:2.2`entry into [ansible/files/runtimes.json](https://github.com/apache/incubator-openwhisk/blob/master/ansible/files/runtimes.json#L209)
- Update and add doc file into [docs/action_node.md](https://github.com/apache/incubator-openwhisk/tree/master/docs)
- Update unicode tests https://github.com/apache/incubator-openwhisk/tree/master/tests/dat/actions/unicode.tests

### 3. Add `dotnet:2.2` to docker-compose deploy
- Add `dotnet:2.2` into runtime manifest
https://github.com/apache/incubator-openwhisk-devtools/blob/master/docker-compose/docker-whisk-controller.env#L64

### 4. Add `dotnet:2.2` to deploy-kube
- Add `ndotnet:2.2` into runtime manifest
https://github.com/apache/incubator-openwhisk-deploy-kube/blob/master/helm/openwhisk/runtimes.json

### 5. Update wskdeploy with `dotnet:2.2`
- Update spec, and tests for `dotnet:2.2`
https://github.com/apache/incubator-openwhisk-wskdeploy/blob/master/runtimes/runtimes.go#L267
https://github.com/apache/incubator-openwhisk-wskdeploy/blob/master/specification/html/spec_actions.md#valid-runtime-names
https://github.com/apache/incubator-openwhisk-wskdeploy/blob/master/tests/src/integration/runtimetests/manifest.yaml#L5

### 6 Update website with `dotnet:2.2`
- Update js action section with `dotnet:2.2`
https://github.com/apache/incubator-openwhisk-website/blob/master/_layouts/documentation.html
https://github.com/apache/incubator-openwhisk-website/tree/master/_includes/code

**Comment by csantanapr** [comment]
@shawnallen85  is going to be working on this

**Comment by rabbah** [comment]
@csantanapr why does wskdeploy needs the manifest - this is a bug I'm raised before. wskdeploy should read the manifest against its api host. 

@pritidesai @mrutkows ?

**Comment by rabbah** [comment]
@csantanapr @mrutkows have you thought about automating 6? just pull the file from this repo so we have one less copy/place to edit, and auto generate some of the HTML.

**Comment by rabbah** [comment]
@ddragosd do you think we should auto generate the JSON in the makefile so we avoid modifying the downstream repo for every new runtime change here?

**Comment by csantanapr** [comment]
I agree in general @rabbah we should re-use the runtimes.json and/or the api host and avoid duplication.

**Comment by csantanapr** [comment]
https://github.com/apache/incubator-openwhisk/pull/4172

**Comment by csantanapr** [comment]
All places are now updated.
I found 2 small items to fix and submitted PRs:
- https://github.com/apache/incubator-openwhisk-devtools/pull/186
- https://github.com/apache/incubator-openwhisk/pull/4180
